test/integration/garbage_collector_test.go

```diff
@@ -279,3 +283,86 @@ func TestCreateWithNonExisitentOwner(t *testing.T) {
 		t.Fatalf("deleted unexpected pod: %v", *deletedPod)
 	}
 }
+
+func createRCsPods(t *testing.T, clientSet clientset.Interface, id int, rcUIDs []types.UID, wg *sync.WaitGroup) {
+	defer wg.Done()
+	rcClient := clientSet.Core().ReplicationControllers(framework.TestNS)
+	podClient := clientSet.Core().Pods(framework.TestNS)
+	rcName := toBeDeletedRCName + strconv.Itoa(id)
+	toBeDeletedRC, err := rcClient.Create(newOwnerRC(rcName))
+	if err != nil {
+		t.Fatalf("Failed to create replication controller: %v", err)
+	}
+	rcUIDs[id] = toBeDeletedRC.ObjectMeta.UID
+	// these pods should be cascadingly deleted.
+	for j := 0; j < 3; j++ {
+		podName := garbageCollectedPodName + strconv.Itoa(id) + strconv.Itoa(j)
```

Comment:
(I know you don't use numbers that high but it's not good to leave landmines like this)